WinUIGallery/PageHeader.xaml

```diff
@@ -63,6 +63,11 @@
                     Text="{x:Bind Title, Mode=OneWay}"
                     TextTrimming="CharacterEllipsis"
                     TextWrapping="NoWrap" />
+                <TextBlock x:Name="NamespaceTextBlock" Grid.Row="1" Text="{x:Bind ApiNamespace, Mode=OneWay}"
+                    Visibility="{x:Bind ApiNamespace, Converter={StaticResource emptyStringToVisibilityConverter}, Mode=OneWay}"
```

Comment:
Visibility is only needed if namespace is optional, which I don't think it should be.

Reply:
There are currently three items for where we hit this:
* DataGrid
* Compact Sizing
* XAMLCompInterop

I didn't know what the best namespaces are which is why they are left blank. If we find good names for those things and also keep in mind future sample pages we might create,, we can surely delete that.